src/route/SocialworkerRoute.tsx

```
@@ -39,9 +44,10 @@ const SocialworkerRoute = () => {
         <Routes>
           <Route path="/" element={<SocialworkerHomePage />} />
           <Route path="/my/*" element={<SocialworkerMyRoute />} />
+          
```

ESLint

[error] Delete `··········`

(prettier/prettier)